frontend/themes/README

```diff
@@ -0,0 +1,3 @@
+Place your theme folder in this folder.
+
+See https://vaadin.com/docs-beta/latest/theming/application-theme/ for getting started on theming Vaadin Flow applications.
```

Comment:
I think this could have been added as a comment to the beginning of `styles.css` instead saying that "this file and component specific files are automagically included"

Reply:
Updated. I'll update to other projects as well if it is fine this way.